config/custom-environment-variables.yaml

```diff
@@ -5,3 +5,14 @@ dm_upload_url: DM_UPLOAD_URL
 dm_find_documents_by_creator_url: DM_OWNED_URL
 dm_find_documents_by_metadata_url: DM_SEARCH_URL
 em_viewer_url: DM_VIEWER_URL
+dm_url: http://localhost:4603
+an_url: http://localhost:4621
+idam:
+  base_url: IDAM_BASE_URL
+  s2s_url: IDAM_S2S_URL
```

Comment:
can this please be 
`IDAM_USER_BASE_URI`
`IDAM_S2S_BASE_URI`
to keep consistent throughout our projects please